cloudinit/util.py

```diff
@@ -1765,7 +1766,16 @@ def get_config_logfiles(cfg):
             logs.append(target)
         elif ["tee", "-a"] == parts[:2]:
             logs.append(parts[2])
-    return list(set(logs))
+
+    # add rotated log files
+    for _log in logs:
```

Comment:
Nit: free variables prepended with an underscore mean unused variables in this codebase, a la Rust. `_log` and `_rotated_logfile` do not collide with any Python keyword nor anything else, if that would be the case we could rename them or append an underscore. I wonder if it would be more readable to just rename them as `log` and `rotated_logfile`.

Reply:
renamed vars to be compliant with codebase